pkg/model/resource/options_test.go

```diff
@@ -17,10 +17,9 @@ var _ = Describe("Resource Options", func() {
 			Expect(options.Validate()).To(Succeed())
 		})
 
-		It("should fail if the Group is not specified", func() {
+		It("should succeed if the Group is not specified", func() {
 			options := &Options{Version: "v1", Kind: "FirstMate"}
-			Expect(options.Validate()).NotTo(Succeed())
-			Expect(options.Validate().Error()).To(ContainSubstring("group cannot be empty"))
+			Expect(options.Validate()).To(Succeed())
```

Comment:
if is v2 the "group cannot be empty" otherwise yes. we need to improve the check

Reply:
Can't check the project version here, as to check that func IsV2() or IsV3() is part of project config which is not available here at unit test level.

Reply:
we need to have the ValidateV2 test and the Validate.
When we no longer support V2 this test and func will be removed. However, so far we need to ensure that has no breaking changes.

Reply:
Done

Reply:
You can close this one it is solved 👍